xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/internal/filter/ImageFilter.java

```diff
@@ -179,8 +179,19 @@ private String getFileName(Attr source) throws MimeTypeException
             int separator = value.lastIndexOf('/');
             fileName = separator < 0 ? value : value.substring(separator + 1);
             try {
+
+                // the + character is not escaped in HtmlCleaner as it is an authorized character
+                // but when decoding it for URL it would be replaced by a space
+                // so we encode it manually here.
+                fileName = fileName.replaceAll("\\+", "%2B");
```

Comment:
This does not make sense to me. If the URL contains a + then it should be a white space otherwise it means there is a bug in whatever generated this URL.

Reply:
I agree that generally speaking we shouldn't need to do that. But here again this solution is specific to office converter: I don't know if it's a bug from jodconverter or if it's directly libre office export engine, but the fact is it creates URL such as:

`<img src="Test_accents%20&%20%C3%A9$%C3%B9%20%3C!-_.+*'()%3B%3F:@=_html_b0c95044fd2eb8b7.gif" align="bottom"/>` where `+` means `+`. I don't know why it's not encoded. 

I'll open an issue on jodconverter to see if we can improve that in jodconverter.

Reply:
FTR I opened an issue in jodconverter about this: https://github.com/sbraconnier/jodconverter/issues/125 I'll add a comment with a todo to fix that once this issue is closed.